public/css/breadcrumb-nav.css

```
@@ -0,0 +1,229 @@
+
+  .nav-trigger-zone.touched + .breadcrumb-nav,
+  .breadcrumb-nav.touched {
+    transform: translateY(100px);
+  }
+}
+
+/* ============================================ */
+/* ANIMATION FOR ENTRANCE */
+/* ============================================ */
+
+@keyframes slideDownNav {
```

Stylelint (17.4.0)

[error] Expected keyframe name "slideDownNav" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)